fabric-containers-v0/src/main/java/net/fabricmc/fabric/impl/client/container/ScreenProviderRegistryImpl.java

```diff
@@ -68,7 +67,7 @@ public void registerFactory(Identifier identifier, ContainerFactory<AbstractCont
 	}
 
 	public static void init() {
-		ClientSidePacketRegistry.INSTANCE.register(PacketTypes.OPEN_CONTAINER, (packetContext, packetByteBuf) -> {
+		Client.CLIENT_PLAY.register(ContainerProviderImpl.OPEN_CONTAINER, (packetContext, packetByteBuf) -> {
```

Comment:
Im not a fan of just having `Client` here, when reading the code it doesnt really say that its networking stuff.

`PacketReceiverRegistry.Client` might be too long though. What are you thoughts on this?

Reply:
I do need suggestions for holding some fields that have client-only classes in their signature. currently i use subclasses to achieve lazy classloading to prevent loading client classes. Splitting them into everything for client/server is viable as well.